extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -196,6 +212,45 @@ export class Placement {
     });
   }
 
+  /**
+   * Gets instructions for the placement in terms of height, width and margins.
+   * If responsive is on, ad should be placed at full viewport width and a
+   * proportionate height, and the margins are adjusted so that the ad edges
+   * stick to both ends of the viewport.
+   * @param {!./ad-network-config.SizeInfoDef} sizing
+   * @return {PlacementDef}
+   * @private
+   */
+  getPlacement_(sizing) {
```

Comment:
See comment above, but I think something like getPlacementSizing would be a better name.

Reply:
Done